src/processors/processVrmRequest.ts

```diff
@@ -0,0 +1,27 @@
+import { APIGatewayProxyEvent } from 'aws-lambda';
+import { getUserDetails } from '../services/user';
+import { TechrecordGet } from '../models/post';
+
+export const processPatchVrmRequest = (currentRecord: TechrecordGet, event: APIGatewayProxyEvent): Array<TechrecordGet> => {
+  const userDetails = getUserDetails(event.headers.Authorization!);
```

Comment:
I'd rather ?? '' and then let the method fail than non null assert, same as below